File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
int n, c, r;

{
    //int n = get_int("Height: ");
}

do
{

    n = get_int("Height: ");

}
while (n < 1 || n >= 8);

for (r = 0; r < n; r++)
{
    for (c = 0; c > n; c++)
    {
        printf("#");
    }
    printf("\n");
}

}

File: sentimental-mario-less/mario.py
from cs50 import get_int

# Get_int for int between 1 and 8

while True:
    h = get_int ("Height: ")
    if n >= 1 and n <=8:
        break

for i in range(h):
    for j in range(h):
        if i + j >= h - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print()



